ui/server/src/main/scala/pl/touk/nussknacker/ui/config/processtoolbar/ProcessToolbarConfig.scala

```diff
@@ -0,0 +1,61 @@
+package pl.touk.nussknacker.ui.config.processtoolbar
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.LazyLogging
+
+import java.util.UUID
+
+case class ProcessToolbarsConfig(uuid: Option[UUID], topLeft: List[ToolbarPanelConfig], bottomLeft: List[ToolbarPanelConfig], topRight: List[ToolbarPanelConfig], bottomRight: List[ToolbarPanelConfig])
+
+object ProcessToolbarsConfigProvider extends LazyLogging {
+
+  import pl.touk.nussknacker.engine.util.config.CustomFicusInstances._
+  import net.ceedubs.ficus.readers.ArbitraryTypeReader._
+  import net.ceedubs.ficus.readers.EnumerationReader._
+
+  private val emptyConfig: Config = ConfigFactory.parseString(
+    """
+      |{
+      |  processToolbarConfig {
+      |    defaultConfig {
+      |      topLeft: []
+      |      bottomLeft: []
+      |      topRight: []
+      |      bottomRight:[]
+      |    }
+      |  }
+      |  categoryConfig: {}
+      |}
+      |""".stripMargin
+  )
+
+  private val defaultProcessToolbarConfigPath = "processToolbarConfig.defaultConfig"
+  private val categoryProcessToolbarConfigPath = "processToolbarConfig.categoryConfig"
+
+  /**
+    * TODO: Merging arrays in typesafe Config is done in primitive way - figure how to better do merging configs..
+    */
+  def create(config: Config, category: Option[String]): ProcessToolbarsConfig = {
+    if (!config.hasPath(defaultProcessToolbarConfigPath)) {
```

Comment:
It's jus in case when there is no configuration.. But if we have some base config in `deafultUiConfig.conf` then this situation will never happen? So, I will remove it.